src/index.html

```diff
@@ -29,8 +30,8 @@ <h2 class= 'user-info_title'>Your Info</h2>
         </article>
       </div> 
       <div class='user-info-area'>
+<!-- Activity Widgets  -->
         <div class='activity genre'>
-          <!-- <h2 class= 'activity-data_title'>Activity</h2> -->
           <article class='activity-data-today widget'>
             <h3>Activity Today</h3>
             <section class='activity-data_activity-minutes'>
```

Comment:
one easy way we can reduce `<div>` use is by using `<canvas>` elements instead of `<section>` for each chart. This is the element type required by Chart.js. Then we can go make many of these `<div>` elements `<section>` elements.